ballista/scheduler/ui/src/setupProxy.js

```diff
@@ -0,0 +1,14 @@
+// Proxy configuration for the development server.
```

Comment:
@simicd  We need to add the ASF header here

Reply:
Thanks for the review @andygrove! Added the header and also removed unused import that caused ESLint to raise a warning - since this is my first PR in the arrow-balista repo it requires you to trigger the CI